.ci/Jenkinsfile

```diff
@@ -148,3 +154,25 @@ def cleanup(){
   }
   unstash 'source'
 }
+
+def pushDockerImage(){
+  dir("${BASE_DIR}"){
+    dockerLogin(secret: "${env.DOCKER_REGISTRY_SECRET}",
+      registry: "${env.DOCKER_REGISTRY}")
+    sh(label: 'Build Docker image',
+      script: """docker build \
+        -t ${env.DOCKER_IMG_TAG} \
+        --label BRANCH_NAME=${env.BRANCH_NAME} \
+        --label GIT_SHA=${env.GIT_BASE_COMMIT} \
+        --label GO_VERSION=${env.GO_VERSION} \
+        --label TIMESTAMP=\$(date +%Y-%m-%d_%H:%M) \
```

Comment:
nice! I didn't release labels can be added without changing the Dockerfile itself.